config/oracle/odbcinst.ini

```diff
@@ -1,3 +1,3 @@
 [Oracle-Driver]
 Description = Oracle ODBC driver for Oracle 12.2
-Driver = /opt/oracle/instantclient_12_2/libsqora.so.12.1
+Driver = /opt/oracle/instantclient_19_6/libsqora.so.19.1
```

Comment:
it should point to /opt/oracle/instantclient/libsqora.so to make it even more generic